frontend/src/components/collaborator/CollaboratorListItem.vue

```diff
@@ -1,104 +1,77 @@
+                <template v-for="icon in roles[collaborator.role].icons"
+                  ><v-icon :key="icon" x-small>{{ icon }}</v-icon
+                  >&thinsp;</template
+                ></span
+              >
+            </button>
+          </template>
+          {{ $tc(roles[collaborator.role].abilitiesTranslation) }}
+        </v-tooltip>
       </v-list-item-subtitle>
     </v-list-item-content>
-    <v-list-item-action v-if="collaborator.status === 'invited'" class="ml-2">
-      <icon-button
-        color="normal"
-        icon="mdi-refresh"
-        :animate="resendingEmail"
-        :disabled="disabled"
-        @click="resendInvitation"
-      >
-        {{ $tc('components.collaborator.collaboratorListItem.resendEmail') }}
-      </icon-button>
-    </v-list-item-action>
-    <v-list-item-action class="ml-4">
-      <v-tooltip :disabled="disabled || !isLastManager" top>
-        <template #activator="{ on, attrs }">
-          <div v-bind="attrs" v-on="on">
-            <api-select
-              :value="collaborator.role"
-              :uri="collaborator._meta.self"
-              fieldname="role"
-              :items="[
-                { key: 'member', translation: $tc('entity.camp.collaborators.member') },
-                { key: 'manager', translation: $tc('entity.camp.collaborators.manager') },
-                { key: 'guest', translation: $tc('entity.camp.collaborators.guest') },
-              ]"
-              item-value="key"
-              item-text="translation"
-              :my="0"
-              dense
-              vee-rules="required"
-              :disabled="disabled || isLastManager"
-            />
-          </div>
-        </template>
-        <span>{{
-          $tc(
-            'components.collaborator.collaboratorListItem.cannotAssignAnotherRoleToLastManager'
-          )
-        }}</span>
-      </v-tooltip>
-    </v-list-item-action>
-    <v-list-item-action class="ml-2">
-      <v-tooltip :disabled="disabled || !isLastManager" top>
-        <template #activator="{ on, attrs }">
-          <div v-bind="attrs" v-on="on">
-            <CollaboratorListItemDeactivate :entity="collaborator">
-              <template #activator="{ on: onDialog }">
-                <button-delete
-                  :disabled="(disabled && !isOwnCampCollaboration) || isLastManager"
-                  icon="mdi-cancel"
-                  v-on="onDialog"
-                >
-                  {{ $tc('components.collaborator.collaboratorListItem.deactivate') }}
-                </button-delete>
-              </template>
-            </CollaboratorListItemDeactivate>
-          </div>
-        </template>
-        <span>{{
-          $tc('components.collaborator.collaboratorListItem.cannotRemoveLastManager')
-        }}</span>
-      </v-tooltip>
+    <v-list-item-action class="e-collaborator-item__actions ml-2">
+      <button-edit icon-first color="primary--text" text class="my-n1 v-btn--has-bg" />
```

Comment:
From an accessibility standpoint, using htm5 `disabled="disabled"` is the same as hiding.

If you want the user to see the button which they aren't allowed to use, you should show a normal button and use JS `@click` to display an explanation instead of performing the action.

Reply:
Now I do not show the button at all if the user has no rights to edit.